### ISSUE TITLE
SftpClient: how to resume upload/download? I.e. process SFTP command 'reput' & 'reget' 

### ISSUE THREAD

**Issue by sensboston** [sshnet/SSH.NET#1391]
Does SftpClient supports upload/download resume options, i.e. defined by SFTP protocol specifications commands **reput** & **reget**? 
I need to upload really huge files (from 2GB to 20GB) and these features will be extremely useful.
Can anyone suggest a solution?
Thank you!

**Comment by Rob-Hague** [comment]
I don't think there is anything like that in the library already. I think the closest thing currently is #864

**Comment by sensboston** [comment]
@Rob-Hague, yes, it looks like exactly what I need but PR isn't merged yet 😢 

Also, do you know: is it possible to get the uploaded file MD5 checksum from the remote host?

**Comment by slimequeen3251** [comment]
@Rob-Hague Hi Rob, first of all thank you for the library. Second, I have a question. I am developing a program and it needs resume support as I am dealing with very large files. [WinSCP](https://winscp.net/eng/docs/library) has a dotnet implementation, but not native and is a wrapper. It has resume support. 

I'd very much rather use SSH.NET though in my program. Is it possible to get resume support added soon? As you mentioned above the PR is here: https://github.com/sshnet/SSH.NET/pull/864

 It's been almost a year though I'm not sure how much would need done. 

Thanks again for the great libray!
